Readme.md

```diff
@@ -0,0 +1,32 @@
+# trfind
+
+Trip report search tool used by ClimbPlan, hosted at https://climbplan.com.
+
+# Contributing
+
+This is mostly a weekend project, Pull requests are welcome!
+
+## Wishlist
+
+Ideas are tracked using the (issues page)[https://github.com/thatneat/trfind/issues].
```

Comment:
markdown syntax for the link is backwards!

Reply:
fixed